e2e/time-slider.spec.ts

```diff
@@ -16,7 +16,7 @@ test("is should be possible to use time slider", async ({
     .locator(".MuiSlider-thumb")
     .boundingBox()
     .then((box) => box.x);
-  const firstMark = page.locator(".MuiSlider-mark").first();
+  const firstMark = page.locator(".MuiSlider-mark").nth(2);
```

Comment:
Curious: what is the first mark then ?

Reply:
Aaah, yeah, the name is not correct anymore 😱 With the new CI / CD slider, we hide first and last marks, so we can't target them anymore programmatically.